pkg/manifests/manifests.go

```diff
@@ -69,13 +97,31 @@ func IngressControllerResources(conf *config.Config, self *appsv1.Deployment) []
 	return objs
 }
 
+func getExternalDNSDeployment(conf *config.Config, dnsCMHash string) *appsv1.Deployment {
```

Comment:
I'd prefer to strategically set particular flags in the existing DNS deployment rather than forking the args entirely here.

Reply:
I'm not sure what you mean. Do you mean we should have this check (and resulting args) be determined within the newExternalDNSDeployment function? Or do you mean something along the lines of 
- assign initial args
- check if private
- modify individual entries/add necessary entries if so

Reply:
> Do you mean we should have this check (and resulting args) be determined within the newExternalDNSDeployment function?

Yes. 🙃